manifests/s/shagrath/PS3MediaServer/1.90.1/shagrath.PS3MediaServer.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: shagrath.PS3MediaServer
+PackageVersion: 1.90.1
+PackageLocale: en-US
+Publisher: shagrath

```

```suggestion
Publisher: PS3 Media Server
```
This must match Control Panel.